### PR TITLE
[CARBONDATA-580] Supporting  Spark 2.1 in Carbon

### PR DESCRIPTION
**Pull request by ravipesala**
Supporting  Spark 2.1 in Carbon

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/388/

File: integration/spark2/src/test/scala/org/apache/spark/sql/common/util/QueryTest.scala [hunk 1]
```diff
@@ -41,18 +41,11 @@ class QueryTest extends PlanTest {
...
-  val storeLocation = s"$rootPath/examples/spark2/target/store"
-  val warehouse = s"$rootPath/examples/spark2/target/warehouse"
-  val metastoredb = s"$rootPath/examples/spark2/target/metastore_db"
+  val storeLocation = s"$rootPath/integration/spark2/target/store"
+  val warehouse = s"$rootPath/integration/spark2/target/warehouse"
+  val metastoredb = s"$rootPath/integration/spark2/target/metastore_db"
 
   val spark = {
-    // clean data folder
-    if (true) {
```
Comment (chenliang613) [inline integration/spark2/src/test/scala/org/apache/spark/sql/common/util/QueryTest.scala]: Why remove "clean data folder" operations?

Comment (ravipesala) [inline integration/spark2/src/test/scala/org/apache/spark/sql/common/util/QueryTest.scala]: It is not required now as drop table support is added in carbon spark 2 integration so drop table command can delete the store as well.

Comment (chenliang613) [inline integration/spark2/src/test/scala/org/apache/spark/sql/common/util/QueryTest.scala]: ok

**Pull request closed without merging**